CXP-811: add documentation about new Product ACLs (#527)
akeneo/pim-api-docs: content/rest-api/permissions.md

:::

:::info

> **Up to the 5.0 version**, users with `Overall Web API` were able to make requests on products and product models without additional permission.

Author: Don't worry, for the english, I will validate with Camille.

> **Up to the 5.0 version**, users with `Overall Web API` were able to make requests on products and product models without additional permission.

Reviewer: ```suggestion
**Until the 5.0 version**, users with `Overall Web API` were able to make requests on products and product models without additional permissions.
```

> On SaaS and newer versions, new ACLs were introduced: `Create and update products`, `List products` and `Delete products`.

Reviewer 2: ```suggestion
We introduced the following ACLs on **Akeneo SaaS versions** for more security: `Create and update products`, `List products` and `Delete products`.
```

Reviewer 2: I have a proofreading timeslot on Thursday with Camille. I'll check with her if that's ok to change this sentence. I'd like to add the Security argument to explain why we made it.

> Moreover, in the Enterprise Edition, the [Catalog Permissions](/documentation/permissions.html#catalog-permissions-ee-only) are still applied on the products and product models, on addition to those ACLs.

Reviewer 3: ```suggestion
Moreover, in the Enterprise Edition, the [Catalog Permissions](/documentation/permissions.html#catalog-permissions-ee-only) are still applied on the products and product models, in addition to those ACLs.
```

Reviewer: +1
